### PR TITLE
add a new endpoint to return client config

### PR DESCRIPTION
**Pull request by wei-lee**
To verify:

1. Make sure you have an openshift cluster running somewhere, and create a new project.
2. Login to the cluster using `oc`
3. Provision the keycloak operator using the templates provided in this PR:
    ```
    oc create -f ./deploy/services/keycloak/rbac.yaml
    oc create -f ./deploy/services/keycloak/operator.yaml
    oc create -f ./deploy/services/keycloak/install.yaml
    ```
4. You should see a keycloak instance will be provisioned.
5. Check the README file for setup the development environment. 
6. Go to the MDC and create a new app. Go to the services page and bind the app to IDM.
7. You should see the content of `mobile-services.json` file updated once the app is bound to the service.

### CODE REVIEW

**Comment by wei-lee** [comment]
@craicoverflow This is ready for review now.

**Comment by jhellar** [comment]
Tested on RHPDS integreatly cluster. Followed the verification steps and was able to bind and unbind SSO.

**Comment by craicoverflow** [comment]
@wei-lee 

> Make sure you have an openshift cluster running somewhere, and create a new project.

Does this mean I need a remote cluster running? Should this work with locally with Minishift or no?

**Comment by wei-lee** [comment]
@craicoverflow It should work with minishift too.

**Comment by craicoverflow** [comment]
I've reviewed code, will verify on an RHPDS cluster now.

File: src/reducers/apps.js [hunk 1]
```diff
@@ -35,4 +43,36 @@ const apps = resourceReducer({
...
-export default apps;
+const appConfigs = (state = defaultAppConfigs, action) => {
+  switch (action.type) {
+    case APP_CONFIG_REQUEST:
+      return {
+        ...state,
+        isFetching: true
+      };
+    case APP_CONFIG_SUCCESS: {
+      const appname = action.result.clientId;
```
[the remark this answers is not in the document]
Comment (wei-lee) [reply src/reducers/apps.js]: sure

**Pull request merged**